### DISCUSSION TITLE
🇵🇸  CodeceptJS #FreePalestine

### DISCUSSION THREAD

**Discussion by imranselfy** [codeceptjs/CodeceptJS#4287] [Ideas]
The International Court of Justice (ICJ) has ordered Israel to take all measures within its power to prevent acts that could amount to genocide against Palestinians in the Gaza Strip.2 The court's provisional measures ruling means that Israel is credibly accused of committing genocide in Gaza and must take measures to prevent further damage while the case is ongoing.

Could we also have a message in support of Palestine too?

**Comment by DavertMik** [comment]
Sorry, I'm not from Palestine and not related to that conflict and I don't have an opinion on that. I'm from Ukraine and I suffer from unprovoked war from russian federation. For almost 3 years russia bombs our cities, kills our people. I don't want to measure who suffers more. But all world conflicts are connected, and stopping one can help resolving another one. 

My personal message is to Stop War in Ukraine. As I'm currently staying in Kyiv and regularly wake up from sounds of bombing.
If this project will have active Palestinian contributors, I can add the banner to share their statement as well.

**Reply by imranselfy** [reply]
Palestines are being provoked for the last 75 years..... https://www.statista.com/chart/16516/israeli-palestinian-casualties-by-in-gaza-and-the-west-bank/

This does not even include to 30,000 of the last few months.
![image](https://github.com/codeceptjs/CodeceptJS/assets/114496230/13b30a9c-9a3e-4e86-864f-bb485b607eac)
